exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -126,9 +128,9 @@ public ResultCode export(Collection<SpanData> spans) {
       // for now, there's nothing to check in the response object
       //noinspection ResultOfMethodCallIgnored
       stub.postSpans(request);
-      return ResultCode.SUCCESS;
+      return Futures.immediateFuture(ResultCode.SUCCESS);
```

Comment:
if we go with this PR, we should definitely create follow-on issues to make the exporter implementations actually do their work async.